fem/bilininteg.hpp

```diff
@@ -1679,6 +1683,11 @@ class DiffusionIntegrator: public BilinearFormIntegrator
    int dim, ne, dofs1D, quad1D;
    Vector pa_data;
 
+#ifdef MFEM_USE_CEED
+   // CEED extension
+   CeedData* ceedDataPtr;
```

Comment:
It seems like this struct is never destroyed. Am I missing something?

Reply:
It seem like a good idea to add a constructor (initializing an empty object) and a destructor for `CeedData`.